css/selectors/-ms-clear.json

```diff
@@ -0,0 +1,56 @@
+{
+  "css": {
+    "selectors": {
+      "-ms-clear": {
+        "__compat": {
+          "description": "<code>::-ms-clear</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::-ms-clear",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": "10",
+              "notes": "In an <code><input type='text'></code> element styled with <code>text-align: right</code>, if the clear button is shown, it will clip off the right edge of the text value of the <code><input type='text'></code> element. A workaround is to hide the clear button using <code>display: none</code>."
```

Comment:
I think this ought to be `<code>&lt;input type='text'&gt;</code>` (two times).

Reply:
Resolved 👍